utils/report_block_usage.py

```diff
@@ -1,7 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```

Comment:
```suggestion
# SPDX-License-Identifier: ISC

```

Reply:
This is not allowed by our format checking tool and CI would fail on that

Reply:
The format checking tool is complaining about a newline between the copyright header and the module docstring?

Reply:
Yes, it looks like newline is allowed only between the beginning of the actual code and copyright header or docstring+copyright